### PR TITLE
Remove unneeded greps

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

**Comment by czerw** [comment]
Typo: unnneded, could you please do verification run ?

File: lib/apparmortest.pm [hunk 1]
```diff
@@ -123,10 +123,10 @@ sub setup_mail_server_postfix_dovecot {
 
     # Setting hostname and domain
     if (is_tumbleweed) {
-        $ip = script_output("ip address | grep inet | grep /24 | awk '{ print \$2 }' | cut -d '/' -f1");
+        $ip = script_output(q(ip address | awk '/inet/ && /\/24/ { print $2 }' | cut -d '/' -f1));
```
Comment (baierjan) [inline lib/apparmortest.pm]: ``ip address | awk '/inet/ && /\/24/ { split($2, addr, "/"); print addr[1] }'`` ?

**Comment by mimi1vx** [comment]
> Typo: unnneded, could you please do verification run ?

why ?

**Comment by baierjan** [comment]
> could you please do verification run ?

The functionality seems not to be changed by rewriting greps & cuts inside `awk`

**Pull request merged**